### DISCUSSION TITLE
tips: insert happened to many login

### DISCUSSION THREAD

**Discussion by fengzk-edgeray** [taosdata/TDengine#7741] [Q&A]
![image](https://user-images.githubusercontent.com/89452329/131477030-0cc385ae-b055-470c-8b47-1862780b8916.png)
![image](https://user-images.githubusercontent.com/89452329/131475644-e46c66e5-411e-4b2d-bc5d-717855751cc6.png)
background：
![image](https://user-images.githubusercontent.com/89452329/131476570-e6701168-009c-41e1-a8cb-a2407b8b8483.png)
1. connections to database  using go-driver
2. insert to many values using stable
3. tdengine version 2.0.20.12

**Comment by yu285** [comment]
could you provide the version of go-driver 
and program you used to insert  data.
we need to reproduce it
